### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_ConnectionDetails_ScreenReader: Status message not announced by screen reader for the loading screen after hitting enter on 'connect' button.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9207]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#Win32;#Benchmark; #A11y_SQLAzureDataStudio;#A11ySev2;#MAS4.1.3;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5e5f8423d3249670f55b1466b1980a5f03
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader: NVDA(2019.3.1), JAWS(2020.2001.70)
MAS reference: MAS4.1.3

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Hover over the 'Servers' header and select the 'New Connection' icon.
4. Navigate to connection details.
5. Launch NVDA and navigate till connect button and press enter.
6. Check if proper status message is announced by screen reader.

**Actual:**
Status message about loading content is not announced when we press the connect button on connection details screen.

**Same issue has been observed on following page:
1.Schema Compare->Menu Bar.(After selecting "compare" menu option from menu bar)**

Issue observed in JAWS and NVDA screen readers.
**Expected:**
Status message should be announced for the loading screen that is displayed after we hit connection button.

**User Impact:**
Screen reader users might not come to know what is the exact action being performed after they hit the 'connect' button.

Attachment link reference:
[9207_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4291470/9207_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
dupe of #9410

**Comment by fsteffi** [comment]
GithubTags:#A11y_SQlADS_WCAG2.1;

**Comment by Harika1702** [comment]
<img width="589" alt="image" src="https://user-images.githubusercontent.com/100270074/219294397-0be2a832-7d80-4d41-a6a1-2336504bc3b7.png">
Verified this issue now fixed.
Version: 1.41.1 (user setup)
Commit: 33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:46:39.274Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22623
GitHub Tags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
